x/chainmanager/types/params.go

```diff
@@ -62,6 +65,7 @@ func NewParams(
 // pairs of auth module's parameters.
 // nolint
 func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
+	// TODO fix this
```

Comment:
@murthyvitwit we might need to fix this in this PR

Reply:
Okay, I will look into this.

Reply:
@anilCSE I have added `ParamSetPairs` and basic validation. Suggest me better validation cases if any.